src/components/groups/group.service.ts

```diff
@@ -4,7 +4,15 @@ import { Group } from './group'
 import { User } from '../users/user'
 
 export const getGroups = async (req: Request, res: Response, next: NextFunction) => {
-  req.groups = await Group.query().orderBy('createdAt', 'DESC')
+  const page = req.query.page ? parseInt(req.query.page) : 0
+  const limit = req.query.limit ? parseInt(req.query.limit) : 5
```

Comment:
Since we use TypeScript 3.8, you can use the new [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) operator here when using default values. Note that it'll only use the right hand side if the left side is `null` or `undefined`

Reply:
Applied the suggestions, thanks for the tips!